packages/terra-clinical-item-view/src/ItemView.jsx

```diff
@@ -190,6 +205,14 @@ const ItemView = ({
 
 ItemView.propTypes = propTypes;
 ItemView.defaultProps = defaultProps;
+
+const opts = {
+  Layouts: layouts,
+  EmphasisTypes: emphasisTypes,
+  Alignments: alignments,
```

Comment:
I am thinking these enum's names should be matching the name of the prop they are correlated with. For example:` Alignments` to `AccessoryAlignments`

Reply:
This was done in https://github.com/cerner/terra-clinical/pull/510/commits/3bbac786827c62570528d93a7eae6ac15faf5314